### PR TITLE
Fix bugs related to icalevents update and organization changes

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ requests>=2.20.0
 python-Levenshtein>=0.12.0
 pytz>=2018.9
 whatthepatch>=0.0.5
-icalevents==0.1.25
+icalevents==0.1.24
```
Comment (marco-c) [inline requirements.txt:12]: What is the bug in 0.1.25?

File: auto_nag/iam.py [hunk 2]
```diff
@@ -164,8 +164,13 @@ def get_phonebook_dump(output_dir=""):
 
         new_data[mail] = new
 
-    for person in new_data.values():
+    to_remove = []
+    for mail, person in new_data.items():
         manager_mail = person["manager"]["dn"]
+        if manager_mail not in all_cns:
+            # no manager
+            to_remove.append(mail)
```
Comment (marco-c) [inline auto_nag/iam.py:172]: You're appending to the list but never using it?

File: requirements.txt (hunk 1 above)
Comment (calixteman) [reply requirements.txt:12]: Some problems when comparing datetime with and without timezone.

Comment (marco-c) [reply requirements.txt:12]: Sun Apr 15 1707 03:14:15

Comment (calixteman) [reply requirements.txt:12]: ahah

**Review by marco-c — APPROVED** [review]

**Pull request merged**